### DISCUSSION TITLE
I want a mode that executes inspection only for files that have changed in Pull request

### DISCUSSION THREAD

**Discussion by be-hase** [JetBrains/Qodana#96] [Ideas]
A similar option is `--changes`.
For example, if we specify like this, how about executing only the changed file with the main branch? (for Pull Request)
`--changes --target-branch main` 

For large projects, it takes a long time to execute qodana, so it would be nice to have such an option.

**Comment by brichbash** [comment]
Hi,

Thank you for the request!
I've created an issue [on our tracker](https://youtrack.jetbrains.com/issue/QD-3416/Add-an-option-to-analyze-only-Pull-Request-changes). If you have a spare minute, please vote for it to get notifications when there are any updates.
